src/axom/primal/operators/is_convex.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef AXOM_PRIMAL_IS_CONVEX_HPP_
+#define AXOM_PRIMAL_IS_CONVEX_HPP_
+
+#include "axom/primal/geometry/Segment.hpp"
+#include "axom/primal/geometry/Polygon.hpp"
+#include "axom/primal/operators/orientation.hpp"
+
+namespace axom
+{
+namespace primal
+{
+/*!
+ * \brief Determines if a polygon defined by ordered vertices is convex
+ * 
+ * \param [in] poly The polygon
+ * 
+ * Uses algorithm adapted from [Maa 99], which uses dot products to detect
```

Comment:
Please expand the reference to `[Ma 99]`.

I think it is
> Y. L. Ma, W.T. Hewitt. "Point inversion and projection for NURBS curve and surface: Control polygon approach"
Computer Aided Geometric Design 20(2):79-99, May 2003.

Is that right?